alpaca/common/websocket.py

```diff
@@ -26,7 +35,7 @@ def __init__(
         api_key: str,
         secret_key: str,
         raw_data: bool = False,
-        websocket_params: Optional[Dict] = None,
+        websocket_params: WebsocketParams = WebsocketParams(),
```

Comment:
@alessiocastrica can we please consider this change in a different PR if this change is no mandatory to fix the issue?

This websocket_params is also used from `CryptoDataStream` and `StockDataStream`. We may need to consider them and also if we use pydantic model, we should cover other parameters of `websockets.connect`. However, I think following parameters of different library is tough. So, would like to discuss in a separate PR.

Reply:
Agree with you, I'll remove it as it's not relevant to the issue